src/main/kotlin/com/software/modsen/driverservice/model/Car.kt

```diff
@@ -0,0 +1,43 @@
+package com.software.modsen.driverservice.model
+
+import com.software.modsen.driverservice.dto.response.CarResponse
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter
+import lombok.Setter
+import org.hibernate.annotations.SQLDelete
+
+@Entity
+@Table(name = "cars")
+@SQLDelete(sql = "UPDATE cars SET is_deleted = true WHERE car_id=?")
+@Setter
+data class Car(
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "car_id")
+    var carId: Long?,
+    @Column(name = "color")
+    var color: String,
+    @Column(name = "model")
+    var model: String,
+    @Column(name = "license_plate")
+    var licensePlate: String,
+    @Column(name = "year")
+    var year: Int,
+    @Column(name = "is_deleted")
+    var isDeleted: Boolean = false
+)
+
+fun Car.toCarResponse(): CarResponse {
+    return CarResponse(
+        carId = this.carId,
+        color = this.color,
+        model = this.model,
+        licensePlate = this.licensePlate,
+        year = this.year
+    )
```

Comment:
Same here. Mapping functions should not be in such places

Reply:
Fixed